src/__tests__/actions/resetAction/__test__/changePasswordActions.test.js

```diff
@@ -0,0 +1,73 @@
+import configureMockStore from "redux-mock-store";
+import thunk from "redux-thunk";
+import fetchMock from "fetch-mock";
+import changePasswordAction from "../../../../actions/resetAction/changePasswordActions";
+import { RESET_PASSWORD_SUCCESS, RESET_PASSWORD_FAIL } from "../../../../actions/types";
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+const { BASE_URL } = process.env;
+
+
+
+describe("change password action", () => {
+  let token
+  let changeUrl
+  beforeEach(()=>{
+   token="nevergiveup";
+   changeUrl=`${BASE_URL}/users/reset-password/change/?token=${token}`;
+    })
+    afterEach(() => {
+      fetchMock.restore();
+});
+  it("handles successful password change", () => {
+    fetchMock.patchOnce(changeUrl, {
+      body: { password: "nevergiveup"},
+      headers: {
+           "content-type": "application/json", 
+           'Authorization': `Bearer ${token}`},
+      
```

Comment:
I think this blankline needs fixing too.

Reply:
Thanks let me fix them

Reply:
I have fixed them now